src/backend-queries.js

```diff
@@ -34,101 +34,120 @@ export async function fetchJson(url, dontCache) {
     let json = {};
     if (response.ok)
       json = await response.json();
-    else
-      console.log(response);
+    else {
+      console.error(response);
+      return;
+    }
 
     // save response to cache
     if (!dontCache)
       window.sessionStorage.setItem(url, JSON.stringify(json));
 
     return json;
   } catch (error) {
-    console.log(error, url);
-    return {};
+    console.error(error, url);
+    return;
   }
 }
 
 // get metagraph
-export function getMetagraph() {
-  return fetchJson(metagraphUrl);
+export async function getMetagraph() {
+  return (await fetchJson(metagraphUrl)) || {};
 }
 
 // get hetio definitions
-export function getHetioDefinitions() {
-  return fetchJson(hetioDefinitions);
+export async function getHetioDefinitions() {
+  return (await fetchJson(hetioDefinitions)) || {};
 }
 
 // get hetio styles
-export function getHetioStyles() {
-  return fetchJson(hetioStyles);
+export async function getHetioStyles() {
+  return (await fetchJson(hetioStyles)) || {};
 }
 
 // get hetmech definitions
 export function getHetmechDefinitions() {
-  return hetmechDefinitions;
+  return hetmechDefinitions || {};
 }
 
 // lookup node by id
-export function lookupNodeById(id) {
-  if (!id)
-    return null;
+export async function lookupNodeById(id) {
+  if (!id && id !== 0)
```

Comment:
Does `id` refer to the `id` field in backend database's `Node` table? If it does, then it will never be 0, so this line can be simplified into:
```
  if (!id)
```

Reply:
It refers to the "hetmech"/under-the-hood `id`. Whatever it is, it can be 0, because I ran into it before with this gene: https://search-api.het.io/v1/nodes/0/

Reply:
Interesting. It probably means that when @dhimmel populated the table, he was not using the default `id` value (which always starts from 1 and increments automatically). @dhimmel, is there any reason why you set the node IDs manually?

@vincerubinetti, I am going to approve this PR for now.

Reply:
> is there any reason why you set the node IDs manually?

See https://github.com/greenelab/hetmech-backend/issues/36. In https://github.com/greenelab/hetmech-backend/pull/40, we switched to using the same node IDs as the Neo4j database uses to keep things simpler. I guess Neo4j starts IDs at zero.

Reply:
> is there any reason why you set the node IDs manually?

See https://github.com/greenelab/hetmech-backend/issues/36. In https://github.com/greenelab/hetmech-backend/pull/40, we switched to using the same node IDs as the Neo4j database uses to keep things simpler. I guess Neo4j starts IDs at zero.

Reply:
@dhimmel, got it, thanks.